tests/data_io/test_core_connect.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for related to the Tamr Core-connect"""
+import tamr_toolbox as tbox
+from tamr_toolbox import utils
+from tamr_toolbox.utils.testing import mock_api
+from tamr_unify_client import Client
+from tests._common import get_toolbox_root_dir
+
+CONFIG = utils.config.from_yaml(
+    get_toolbox_root_dir() / "tests/mocking/resources/toolbox_test.yaml"
+)
+
+DATASET_NAME = "test_core_connect"
+PRIMARY_KEYS = None
+
+enforce_online_test = False
+
+
```

Comment:
You should probably add some tests to ensure that the various error handling you have built in the ingest function actually trigger.

Reply:
Tests are added. One test for raising the error of Tamr version; another test for raising the error when an ingestion fails.